scratch-js/Sprite.mjs

```diff
@@ -187,24 +187,17 @@ export class Sprite extends SpriteBase {
   }
 
   createClone() {
-    const parent = this.parent || this;
     const clone = Object.assign(
-      Object.create(Object.getPrototypeOf(parent)),
-      parent
+      Object.create(Object.getPrototypeOf(this)),
+      this
     );
 
-    clone._project = parent._project;
-
-    clone.triggers = parent.triggers.map(
-      trigger =>
-        new Trigger(
-          trigger.trigger,
-          trigger.options,
-          trigger._script.bind(clone)
-        )
+    clone._project = this._project;
+    clone.triggers = this.triggers.map(
+      trigger => new Trigger(trigger.trigger, trigger.options, trigger._script)
```

Comment:
this is already merged obviously (and an unrelated change to this pr) but should there be a method `trigger.clone()`? might be handy for simplifying stuff like this, getting rid of the need to assume the function signature!

Reply:
Yes. You wanna make a PR? :P